lib/usd/ui/CMakeLists.txt

```diff
@@ -23,23 +23,14 @@ set(CMAKE_AUTOUIC ON)
 add_library(${PROJECT_NAME} SHARED)
 
 add_subdirectory(layerEditor)
+add_subdirectory(importDialog)
```

Comment:
With the addition of the Layer Editor, the "ui" folder now contains more than just the import dialog. So I moved the import dialog code to a sub-folder.